ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -326,8 +326,8 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-fips
-- as: aws-ipi-ovn-hypershift-guest-p2-f7
-  cron: 0 6 */7 * *
+- as: aws-ipi-ovn-hypershift-guest-p3-f21
```

Comment:
p1 = GA feature,
p2 = tech preview
p3 = dev preview
Is hypershift in dev preview ?  Perhaps you want `p1-f21` ?

Reply:
Currently, there is no GA for hypershift now. HyperShift is still in TP and DP.

Reply:
Ack